code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/components/AepImageComposable.kt

```diff
@@ -0,0 +1,46 @@
+/*
+  Copyright 2024 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+*/
+
+package com.adobe.marketing.mobile.aepcomposeui.components
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.DefaultAlpha
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.layout.ContentScale
+import com.adobe.marketing.mobile.aepcomposeui.style.AepImageStyle
+
+/**
+ * A composable function that displays a image element with customizable properties.
+ *
+ * @param content Image to be displayed.
```

Comment:
nit: should this var be renamed as its the BitmapPainter being used rather than the image content?

Reply:
Kept this as `content` to be consistent with other Aep*Composables. Changed the kdoc to reflect that it needs to be a painter resource 👍